### ISSUE TITLE
[ToDo] Add interactivity to playground

### ISSUE THREAD

**Issue by galibhassan** [LRydin/Power-Grid-Frequency#7]
#### Data loading
- let the user choose from few different sample data

#### Statistics
- Mark the `mean` frequency
- Present `standard deviation`

@LRydin you said something about `RMS` and `maxima-minima`. Could you please elaborate here?

#### what else?

**Comment by LRydin** [comment]
Yeah, for the engineers the `maxima-minima` this is called `nadir`. So we just have to find `abs(min)` and `abs(max)` and then `max(abs(min),abs(max))`. This only makes sense if you remove the fixed reference frequency of 50/60 Hz.

This brings me to the next important point: We should remove the 50/60 Hz from the data, so that one can compare, on the same plot, recording form Europe (50 Hz) and from the US (60 Hz).

**Comment by LRydin** [comment]
The other measure is just the integral under the curve, or we can do instead something like `sum(abs(freq))`. This should tell us roughly the deviation from the ref. (50 or 60 Hz)
